gulp/tasks/resume.js

```diff
@@ -1,14 +1,22 @@
 'use strict';
 const
     gulp = require('gulp'),
-    exec = require('child_process').exec;
+    exporter= require('../../lib/exporter');
 
-gulp.task('resume', function (cb) {
-    let command = 'resume serve --silent';
-
-    exec(command, function(err, stdout, stderr) {
-        console.log(stdout);
-        console.log(stderr);
-        cb(err);
+//pull json data from resume.json and render an html and pdf version of resume
+gulp.task('resume', function () {
+    const
+        readPathResume = './app/views/resume.hbs',
+        writePathResume = './public/resume.html';
+    exporter.createResume(readPathResume,writePathResume,function(err){
+        if(err){
```

Comment:
Add spaces between all if/else statements and brackets. You can update your linter to encorce this as well.

`if (err)`

`} else {`